specs/span-batches.md

```diff
@@ -304,6 +312,17 @@ Span-batch rules, in validation order:
     that is invalid or derived by other means exclusively:
     - any transaction that is empty (zero length `tx_data`)
     - any [deposited transactions][g-deposit-tx-type] (identified by the transaction type prefix byte in `tx_data`)
+- Overlapped blocks checks:
```

Comment:
Is this implemented?

Reply:
yes. you can find this code in [the derivation PR](https://github.com/ethereum-optimism/optimism/pull/7621/files#diff-95e3cbbeed2bde6b84fb4510e4735f6f5fde5df97cec98f25b14d413005ce4d8R328)